docs/pages/pack/docs/_meta.json

```diff
@@ -6,5 +6,6 @@
   "features": "Features",
   "comparisons": "Comparisons",
   "benchmarks": "Benchmarks",
-  "migrating-from-webpack": "Migrating from Webpack"
+  "migrating-from-webpack": "Migrating from Webpack",
+  "advanced": "Advanced"
```

Comment:
was this missing before?

Reply:
Yeap, https://github.com/vercel/turbo/tree/a2b4abaea73fda11af8ee6c6dc4b64e436e44ddf/docs/pages/pack/docs